amr-wind/core/field_ops.H

```diff
@@ -295,6 +296,31 @@ inline void lincomb(
     lincomb(dst, a, x, xcomp, b, y, ycomp, dstcomp, numcomp, amrex::IntVect(nghost));
 }
 
+//Set the lower bound for a given scalar field   
+inline void field_lower_bound(Field& field, amrex::Real min_value)
```

Comment:
Convert to templated method so that it works with both `Field` and `ScratchField`

Reply:
Should we consider making this generic to vector and scalar fields? I suppose we typically clip only scalars. Is there a generic way to handle vector and scalar fields? For a vector field would we set a min on all components?

Reply:
Use the following signature to follow other `field_ops`

```C++
template <typename FType>
void field_lbound(FType& field, amrex::Real min_value, int scomp = 0, int ncomp = 1, int nghost = 0)
```

This way the call site can determine how to handle vector fields.